orangecontrib/text/lda.py

```diff
@@ -36,10 +37,11 @@ def __init__(self, text, num_topics=5, callback=None):
 
         self.lda = models.LdaModel(id2word=dictionary, num_topics=self.num_topics)
         done = 0
-        for i, part in enumerate(chunk_list(corpus, 95)):
+        for i, part in enumerate(chunk_list(corpus, 100)):
             self.lda.update(part)
             done += len(part)
-            if callback: callback(95.0*done/len(corpus))
+            if callback:
+                callback(len(part))
```

Comment:
Yup, now it's with "current batch size" since callback is `progressBar.advance`.